plugins/pipelines/test/lib/hooks_test.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../test_helper'
+
+describe 'zendesk hooks' do
+  let(:deploy) { deploys(:succeeded_test) }
+  let(:stage) { deploy.stage }
+  let(:next_stages) { [ stages(:test_production), stages(:test_production_pod) ] }
+  let(:fake_deploy) { Hashie::Mash.new(url: 'abc') }
+
+  describe :after_deploy do
+    it 'kicks off the next stages in the deploy' do
+      stage.update!(next_stage_ids: next_stages.map(&:id))
```

Comment:
if you put this in your fixture it avoids making a query in the test run so the specs run a tiny bit faster